contents.json

```diff
@@ -14464,6 +14464,27 @@
       "license": "other",
       "suggested_by": "@SlayterDev",
       "stars": 8
+    },
+    {
+            "title": "Parse Dashboard for iOS",
+            "category-ids": [
+              "parse",
+              "swift",
+              "core-data"
+            ],
+            "description": "A beautiful moile client for managing your Parse apps while you are on the go! Now you can easily view and modify your data in the same way you would on the offical desktop client.",
+            "source": "https://github.com/nathantannar4/Parse-Dashboard-for-iOS",
+            "screenshots": [
+              "https://github.com/nathantannar4/Parse-Dashboard-for-iOS/blob/master/Screenshots/Server.png&raw=true",
```

Comment:
Images are not coming through, should be something like this

https://raw.githubusercontent.com/nathantannar4/Parse-Dashboard-for-iOS/master/Screenshots/Query.png

Can you update all